src/components/capture/details/index.tsx

```diff
@@ -31,31 +34,47 @@ const LIVE_SPECS_QUERY = `
 `;
 
 function CaptureDetails() {
-    const intl = useIntl();
-    useTitle(
-        intl.formatMessage({
-            id: 'browserTitle.captureDetails',
-        })
-    );
+    useBrowserTitle('browserTitle.captureDetails');
 
+    const navigate = useNavigate();
     const [searchParams] = useSearchParams();
     const pubID = searchParams.get(routeDetails.capture.details.params.pubID);
+    if (!pubID) navigate(routeDetails.dashboard.path);
```

Comment:
This is just an idea... maybe when we pull variable from the URL we just validate it is there (or even matches the pattern we want) and kick folks out if the URL get's messed up. Would also like to have a notification displayed as well - but that can come later.